### ISSUE TITLE
Prefix

### ISSUE THREAD

**Issue by gdelgado** [innovia/s3_download_by_date#4]
if the files are on the root of the S3 bucket and there is no prefix. Should the prefix option be left blanks since its a requirement? I got the error below when trying to download a set of files and I want to know if leaving the prefix blank has something to do with it.

/usr/local/lib/ruby/gems/2.1.0/gems/aws-sdk-v1-1.61.0/lib/aws/core/client.rb:375:in `return_or_raise': AWS::S3::Errors::Forbidden (AWS::S3::Errors::Forbidden)

**Comment by innovia** [comment]
Hi

For now you can pass prefix of empty string '' 

Note that this will scan your entire S3 as it goes through every sub folder and that might take some time

Ami

**Comment by innovia** [comment]
Hi @gdelgado 

I fixed the issue, it now looks like this:


````bash
  s3download --bucket elb-logs  --from 'yesterday at 00:00' --to 'now' --save-to ~/Downloads/
````

You did not pass a --prefix option, this will scan the entire bucket and can be very slow if there too many files in that bucket
If you're trying to script this and bypass this question, use --prefix '' (empty string)
Continue without a prefix? (y/n)

the fixed gem version is 0.2.7

reinstall the gem
